### PR TITLE
codeJE: Add event histo and disable sel7 event selection

### PR DESCRIPTION
**Pull request by jloemker**
Hi @vkucera and @Archita-Dash ,

I set the evSel for jets and tracks in the configurable to false/none and added the vertex Z histograms in a new canvas.

Cheerio,
Johanna

### CODE REVIEW

File: codeJE/Compare.C [hunk 1]
```diff
@@ -20,6 +20,7 @@ Int_t Compare(TString pathFileO2 = "AnalysisResults_O2.root", TString pathFileAl
   // Histogram specification: axis label, AliPhysics name, O2Physics path/name, rebin, log scale histogram, log scale ratio, projection axis
 
   VecSpecHis vecHisEvents;
+  AddHistogram(vecHisEvents, "z_{Event} [cm]", "Events Vertex Distribution", "jet-validation-track-collision-qa/controlCollisionVtxZ", 1, 0, 0);
```
Comment (vkucera) [inline codeJE/Compare.C:23]:
```suggestion
  AddHistogram(vecHisEvents, "#it{z}_{prim. vtx.} (cm)", "Events Vertex Distribution", "jet-validation-track-collision-qa/controlCollisionVtxZ", 1, 0, 0);
```

File: codeJE/Compare.C [hunk 2]
```diff
@@ -37,6 +38,8 @@ Int_t Compare(TString pathFileO2 = "AnalysisResults_O2.root", TString pathFileAl
   //  vecSpecVecSpec.push_back(std::make_tuple("events", vecHisEvents, 4, 2));
   if (options.Contains("jets"))
     vecSpecVecSpec.push_back(std::make_tuple("jets", vecHisJets, 3, 2));
+  if (options.Contains("Events"))
+    vecSpecVecSpec.push_back(std::make_tuple("Events", vecHisEvents, 1, 1));
```
Comment (vkucera) [inline codeJE/Compare.C:42]:
```suggestion
  if (options.Contains("Events"))
    vecSpecVecSpec.push_back(std::make_tuple("Events", vecHisEvents, 1, 1));
```
Please use the existing lines above which are currently commented out.
Also please add the missing spaces around "jets" in the line `Contains("jets")`.

File: codeJE/config_tasks.sh [hunk 3]
```diff
@@ -202,8 +202,9 @@ function MakeScriptPostprocess {
   POSTEXEC="echo Postprocessing"
   # Compare AliPhysics and O2 histograms.
   [[ $DOALI -eq 1 && $DOO2 -eq 1 ]] && {
-    OPT_COMPARE=""
+    OPT_COMPARE=" "
```
Comment (vkucera) [inline codeJE/config_tasks.sh:205]: Revert

File: codeJE/config_tasks.sh [hunk 4]
```diff
@@ -202,8 +202,9 @@ function MakeScriptPostprocess {
...
     [ $DOO2_JET_VALID -eq 1 ] && OPT_COMPARE+=" jets "
+    [ $DOO2_JET_VALID -eq 1 ] && OPT_COMPARE+=" Events "
```
Comment (vkucera) [inline codeJE/config_tasks.sh:207]:
```suggestion
    [ $DOO2_JET_VALID -eq 1 ] && OPT_COMPARE+=" events "
```

File: codeJE/dpl-config.json [hunk 5]
```diff
@@ -445,10 +445,10 @@
     },
     "jet-validation-track-collision-qa": {
         "ptLow": "0.15",
-        "ptUp": "100",
+        "ptUp": "1e15",
         "etaLow": "-0.9",
         "etaUp": "0.9",
-        "evSel": "true",
+        "evSel": "false",
```
Comment (vkucera) [inline codeJE/dpl-config.json:451]: Revert. Event selection should be on by default.

**Review by vkucera — CHANGES_REQUESTED** [review]
Thanks @jloemker , please see my comments.

File: codeJE/dpl-config.json (hunk 5 above)
Comment (jloemker) [reply codeJE/dpl-config.json:451]: But then we have the clear differences in the number of jets (?).

**Review by vkucera — APPROVED** [review]

**Pull request merged**